app/views/declaration/add_person.scala.html

```diff
@@ -44,7 +44,9 @@
                 panel = false,
                 f = f("declaration-addperson"),
                 legend = "declaration.addperson.title",
-                legendHidden = true
+                legendHidden = true,
+                heading2 = true,
+                heading2Message = "declaration.addperson.h2"
             ) {
```

Comment:
heading2 and heading2Message are not clear names for parameter names, if you could think of other names would be nice

Reply:
Done